packages/integration-tests/src/tests/ui/applications/index.test.ts

```diff
@@ -0,0 +1,269 @@
+  expectToOpenNewPage,
+  expectToSaveChanges,
+  expectUnsavedChangesAlert,
+  goToAdminConsole,
+  waitForToast,
+} from '#src/ui-helpers/index.js';
+import { expectNavigation, appendPathname } from '#src/utils.js';
+
+import { initialApp, testApp } from './application-test-cases.js';
+import {
+  expectFrameworkExists,
+  expectToClickFramework,
+  expectToProceedCreationFrom,
+  expectToProceedSdkGuide,
+  expectToProceedAppDeletion,
+  expectFrameworksInGroup,
+} from './helpers.js';
+import {
+  type TypedApplicationCase,
+  typedApplicationCases,
+} from './typed-application-test-cases.js';
+
+await page.setViewport({ width: 1920, height: 1080 });
+
+describe('applications', () => {
+  const logtoConsoleUrl = new URL(logtoConsoleUrlString);
+
+  beforeAll(async () => {
+    await goToAdminConsole();
+  });
+
+  it('navigate to applications page', async () => {
+    await expectNavigation(
+      page.goto(appendPathname('/console/applications', logtoConsoleUrl).href)
+    );
+
+    expect(page.url()).toBe(new URL('/console/applications', logtoConsoleUrl).href);
+  });
+
+  it('the table placeholder should be rendered correctly', async () => {
+    await expect(page).toMatchElement(
+      'div[class$=guideLibraryContainer] div[class$=titleEllipsis]',
+      { text: 'Select a framework or tutorial', timeout: 2000 }
+    );
+
+    await expectFrameworksInGroup(page, 'div[class$=guideGroup]:has(>label)');
+  });
+
+  it('create the initial application from the table placeholder', async () => {
+    await expectToClickFramework(page, initialApp.framework);
+
+    await expectToProceedCreationFrom(page, initialApp);
+
+    await expectToProceedSdkGuide(page, initialApp, true);
+
+    // Details page
+    await expect(page).toMatchElement('div[class$=main] div[class$=header] div[class$=name]', {
+      text: initialApp.name,
+    });
+
+    // Back to application list page
+    await expectNavigation(
+      expect(page).toClick('div[class$=main] a[class$=backLink]', {
+        text: 'Back to Applications',
+      })
+    );
+
+    expect(page.url()).toBe(new URL('/console/applications', logtoConsoleUrl).href);
+
+    /**
+     * Note:
+     * Reload the page to refresh new application data by the SWR,
+     * since test operations is so quick and the SWR is not updated
+     */
+    await page.reload();
+
+    await expect(page).toMatchElement('table div[class$=item] a[class$=title]', {
+      text: initialApp.name,
+    });
+  });
+
+  it('can open the logto github repo issue page when click on the framework not found button', async () => {
+    await expect(page).toClick('div[class$=main] div[class$=headline] button span', {
+      text: 'Create Application',
+    });
+
+    await expectModalWithTitle(page, 'Start with SDK and guides');
+
+    await expect(page).toClick(
+      '.ReactModalPortal div[class$=header] button[class$=requestSdkButton] span',
+      {
+        text: "Can't find your guide?",
+      }
+    );
```

Comment:
Can we find the button without matching button text?

Reply:
Updated.